Megaparsec/Printable.lean

```diff
@@ -98,7 +97,7 @@ instance : Printable String where
 instance : Printable UInt8 where
   showTokens := stringPretty ∘ Functor.map (fun i => Char.ofNat $ i.toNat)
 
-open Bit in
+open ByteArray in
```

Comment:
I assume this change is in Straume too? If yes, then I approve. If no, then I need to understand how are we sure that binary parsing is mutually compatible between our libraries.